cmd/thv/app/secret.go

```diff
@@ -280,3 +292,132 @@ func getSecretsManager() (secrets.Provider, error) {
+		switch input {
+		case "encrypted":
+			providerType = secrets.EncryptedType
+		case "1password":
+			providerType = secrets.OnePasswordType
+		case "none":
+			providerType = secrets.NoneType
+		default:
+			fmt.Println("Invalid provider. Please enter 'encrypted', '1password', or 'none'.")
+			continue
+		}
+		break
+	}
+
+	fmt.Printf("\nYou selected: %s\n", providerType)
+
+	// Provider-specific setup
+	switch providerType {
+	case secrets.EncryptedType:
+		if err := setupEncryptedProvider(ctx); err != nil {
+			return fmt.Errorf("failed to set up encrypted provider: %w", err)
+		}
+	case secrets.OnePasswordType:
+		if err := setupOnePasswordProvider(ctx); err != nil {
+			return fmt.Errorf("failed to set up 1Password provider: %w", err)
+		}
+	case secrets.NoneType:
+		fmt.Println("\nSecrets functionality will be disabled.")
+	}
+
+	if err := SetSecretsProvider(providerType); err != nil {
+		return fmt.Errorf("failed to update configuration: %w", err)
+	}
+
+	fmt.Printf("\n✓ Secrets provider '%s' has been successfully configured!\n", providerType)
+	return nil
+}
+
+func setupEncryptedProvider(ctx context.Context) error {
+	fmt.Println("\nSetting up encrypted secrets provider...")
+	fmt.Println("You will need to provide a password to encrypt your secrets.")
+	fmt.Println("This password will be stored in your OS keyring if available.")
+
+	// Test that we can create the provider (this will prompt for password)
+	fmt.Println("\nTesting encrypted provider setup...")
+	provider, err := secrets.CreateSecretProvider(secrets.EncryptedType)
+	if err != nil {
+		return fmt.Errorf("failed to initialize encrypted provider: %w", err)
+	}
+
+	// Test basic functionality
+	testKey := "setup-test"
+	testValue := "test-value"
+
+	if err := provider.SetSecret(ctx, testKey, testValue); err != nil {
+		return fmt.Errorf("failed to test secret storage: %w", err)
+	}
+
+	retrievedValue, err := provider.GetSecret(ctx, testKey)
+	if err != nil {
+		return fmt.Errorf("failed to test secret retrieval: %w", err)
+	}
+
+	if retrievedValue != testValue {
+		return fmt.Errorf("secret test failed: expected %s, got %s", testValue, retrievedValue)
+	}
+
+	// Clean up test secret
+	_ = provider.DeleteSecret(ctx, testKey)
+
+	fmt.Println("✓ Encrypted provider setup successful!")
+	return nil
+}
+
+func setupOnePasswordProvider(ctx context.Context) error {
+	fmt.Println("\nSetting up 1Password secrets provider...")
+
+	// Check for service account token
+	token := os.Getenv("OP_SERVICE_ACCOUNT_TOKEN")
+	if token == "" {
+		fmt.Println("\nERROR: OP_SERVICE_ACCOUNT_TOKEN environment variable is not set.")
+		fmt.Println("To use 1Password as your secrets provider, you need to:")
+		fmt.Println("1. Create a service account in your 1Password account")
+		fmt.Println("2. Generate a service account token")
+		fmt.Println("3. Set the OP_SERVICE_ACCOUNT_TOKEN environment variable")
+		fmt.Println("\nFor more information, visit: https://developer.1password.com/docs/service-accounts/")
```

Comment:
nit: you can have these (and other places with multiple printlns) a single multiline const string or template.